src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue26049.cs

```diff
@@ -0,0 +1,46 @@
+#if TEST_FAILS_ON_ANDROID && TEST_FAILS_ON_WINDOWS   //More information - https://github.com/dotnet/maui/issues/27494
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue26049 : _IssuesUITest
+	{
+		public Issue26049(TestDevice device) : base(device)
+		{
+		}
+
+		public override string Issue => "[iOS] Fix ShellContent Title Does Not Update at Runtime";
+
+		[Test, Order(1)]
+		[Category(UITestCategories.Shell)]
+		public void VerifyFirstShellContentTitle()
+		{
+			App.WaitForElement("ChangeShellContentTitle");
+			App.Click("ChangeShellContentTitle");
+			VerifyScreenshot();
+		}
+
+		[Test, Order(2)]
+		[Category(UITestCategories.Shell)]
+		public void VerifyNewlyAddedShellContentTitle()
```

Comment:
Pending macOS snapshots. Already available in the latest build:
![image](https://github.com/user-attachments/assets/c2982e7d-f7a9-4e62-aee5-b93f88ee79bc)
Could you commit the images?

Reply:
I have added mac snapshot. Please let me know if you any concerns